pkg/controller/provider/container/ovirt/reconciler.go

```diff
@@ -405,13 +405,13 @@ func (r *Reconciler) refresh(ctx *Context) (err error) {
 			"event",
 			event)
 		var changeSet []Updater
-		changeSet, err = r.changeSet(ctx, event)
```

Comment:
Using `err` within the loop will not result in a deterministic return value since we continue.

Reply:
Great catch